### PR TITLE
If command.Execute() return err, print to stdErr

### PR DESCRIPTION
**Pull request by ghost**
The current kube-proxy not print error if (execute() failed) . not so good and not so friendly to user.
If print err to stdError, will show us why it failed.

### CODE REVIEW

File: cmd/kube-proxy/proxy.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ package main
 
 import (
 	goflag "flag"
+	"fmt"
```
Comment (dixudx) [inline cmd/kube-proxy/proxy.go]: did not run `gofmt` ?

File: cmd/kube-proxy/proxy.go [hunk 2]
```diff
@@ -41,7 +42,8 @@ func main() {
 	logs.InitLogs()
 	defer logs.FlushLogs()
 
-	if err := command.Execute(); err != nil {
+	if err := command.Execute(); err != nil {
```
Comment (dixudx) [inline cmd/kube-proxy/proxy.go]: anything changed on this line?

File: cmd/kube-proxy/proxy.go [hunk 3]
```diff
@@ -41,7 +42,8 @@ func main() {
 	logs.InitLogs()
 	defer logs.FlushLogs()
 
-	if err := command.Execute(); err != nil {
+	if err := command.Execute(); err != nil {		
+		fmt.Fprintf(os.Stderr, "%v\n", err)
```
Comment (dixudx) [inline cmd/kube-proxy/proxy.go]: better to use `fmt.Fprintf(os.Stderr, "error: %v\n", err)`?

`kube-proxy` is not the only one cmd that missing such error msg. Please update them as well.

File: cmd/kube-proxy/proxy.go (hunk 1 above)
Comment (netroby) [inline cmd/kube-proxy/proxy.go]: I resubmit , and run go fmt yet.

File: cmd/kube-proxy/proxy.go (hunk 3 above)
Comment (netroby) [inline cmd/kube-proxy/proxy.go]: Change as you mentioned

**Comment by netroby** [comment]
Hello, @dixudx
I added them as you tip.
And run go fmt for them

**Comment by netroby** [comment]
ping @dchen1107 @lavalamp @mikedanese @thockin

File: cmd/kubelet/kubelet.go [hunk 4]
```diff
@@ -106,6 +106,7 @@ func main() {
 
 	// run the kubelet
 	if err := app.Run(kubeletServer, kubeletDeps); err != nil {
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
```
Comment (lavalamp) [inline cmd/kubelet/kubelet.go]: The die function already does this.

**Comment by lavalamp** [comment]
2/3 of the changes look good, sorry for slow response

**Comment by netroby** [comment]
@lavalamp Hello, remove as you ask

**Comment by lavalamp** [comment]
ok to test

**Pull request merged**